scanpy/preprocessing/_simple.py

```diff
@@ -757,7 +757,9 @@ def scale(
     annotated with `'mean'` and `'std'` in `adata.var`.
     """
     _check_array_function_arguments(layer=layer, obsm=obsm)
-    return scale_array(data, zero_center=zero_center, max_value=max_value, copy=copy)
+    return scale_array(
+        data, zero_center=zero_center, max_value=max_value, copy=copy  # noqa: F821
+    )
```

Comment:
It doesn't know `data`. Even Pycharm flags this. I don't know how such code even ended up in Scanpy.

Reply:
For this review process, if there is code that looks bad I would prefer that you flagged it for removal rather than adding it to an ignore list.

I think this is where a lot of the value of turning `flake8` on will come from.

Reply:
Sure. But then you will have to fix it **now** or flake8 will keep failing :)

Whatever you prefer.

Reply:
but is this the `sc.pp.scale` ?  I wonder if it's tested?